src/extensionsIntegrated/TabBar/more-options.js

```diff
@@ -218,6 +190,59 @@ define(function (require, exports, module) {
         CommandManager.execute(Commands.FILE_REOPEN_CLOSED);
     }
 
+    /**
+     * "RENAME FILE"
+     * This function handles the renaming of the file that was right-clicked
+     *
+     * @param {String} filePath - path of the file to rename
+     */
+    function handleFileRename(filePath) {
+        if (filePath) {
+            // First ensure the sidebar is visible so users can see the rename action
+            CommandManager.execute(Commands.SHOW_SIDEBAR);
```

Comment:
This is a good check, i guess this can be moved to the core itself as part of the underlying command.